fdbserver/ClusterController.actor.cpp

```diff
@@ -2538,6 +2538,10 @@ ACTOR Future<Void> workerAvailabilityWatch(WorkerInterface worker,
 				cluster->removedDBInfoEndpoints.insert(worker.updateServerDBInfo.getEndpoint());
 				cluster->id_worker.erase(worker.locality.processId());
 				cluster->updateWorkerList.set(worker.locality.processId(), Optional<ProcessData>());
+				TraceEvent("ClusterControllerWorkerFailed", cluster->id)
+					.detail("WorkerProcessId", worker.locality.processId())
```

Comment:
Minor nit, I think we can process "Worker*" prefix from all three details. Rest of the code base follows same pattern just naming `ProcessId`, so to be consistent, which also makes writing splunk queries easier.